lib/engine/action/split.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
+  module Action
+    class Split < Base
+      attr_reader :corporation
```

Comment:
why is this needed? just use convert?

Reply:
Use the convert action to be the action that happens when you choose to split a corp in 1871?